client/interview-controller.js

```diff
@@ -0,0 +1,55 @@
+// Description:
+//   A Hubot script which helps hrs to control interviews.
+//
+// Commands:
+//  begin group Huntflow Reloaded
+//    begin admin
+//      hubot удалить интервью - remove an interview from the list
+//    end admin
+//  end group
+//
+
+const routines = require('hubot-routines')
+const utils = require('./utils')
+
+const interviewsInstance = require('./interviews-service')
+
+const candidateRegExp = new RegExp(/([А-Я][а-я]+)\s([А-Я][а-я]+)/g)
```

Comment:
The script will be used with other Hubot scripts so it would be better to make more specific regexp, to avoid mixes (for example `удалить интервью кандидата name`).